### ISSUE TITLE
looping not work

### ISSUE THREAD

**Issue by Araara7** [forforgebe/moonbix#2]
![image](https://github.com/user-attachments/assets/a5ba0840-97e6-4ce9-a06b-a3311cc7192b)

1. when ticket is 0 back to this menu
2. im wating for 1 hour (there is error im forget to screen shot )
3. need operate auto again

**Comment by forforgebe** [comment]
can you check log on logs/app.log?

**Comment by Araara7** [comment]
> can you check log on logs/app.log?
**_this sir history of Log_**
[2024-09-22T09:49:50.681Z] [INFO] [1825229511] user authenticated
[2024-09-22T09:49:50.787Z] [INFO] [1825229511] get info (1 tickets left)
[2024-09-22T09:49:53.025Z] [INFO] [1825229511] game "11671fb7103fe5979b9288d711c6e871" started (please wait 45 seconds)
[2024-09-22T09:50:38.250Z] [INFO] [1825229511] game "11671fb7103fe5979b9288d711c6e871" finished +255 pts [1/1]
[2024-09-22T10:05:00.183Z] [INFO] [1825229511] user authenticated
**[2024-09-22T10:05:03.288Z] [ERROR] [1825229511] Cannot read properties of null (reading 'qualified')** **feedback error from this sir**


**from here im choose auto again**

 
[2024-09-22T10:56:37.337Z] [INFO] [1825229511] user authenticated
[2024-09-22T10:56:37.642Z] [INFO] [1825229511] get info (5 tickets left)
[2024-09-22T10:56:39.917Z] [INFO] [1825229511] game "6bce6b0ce7675fc636941d7c93fdb2fb" started (please wait 45 seconds)
[2024-09-22T10:57:25.413Z] [INFO] [1825229511] game "6bce6b0ce7675fc636941d7c93fdb2fb" finished +325 pts [1/5]
[2024-09-22T10:57:30.727Z] [INFO] [1825229511] game "9f321d06e5d1a439d6bee8788b790bdc" started (please wait 45 seconds)
[2024-09-22T10:58:16.211Z] [INFO] [1825229511] game "9f321d06e5d1a439d6bee8788b790bdc" finished +255 pts [2/5]
[2024-09-22T10:58:21.497Z] [INFO] [1825229511] game "21e282a16806f71607a9a49ed188021d" started (please wait 45 seconds)
[2024-09-22T10:59:06.809Z] [INFO] [1825229511] game "21e282a16806f71607a9a49ed188021d" finished +285 pts [3/5]
[2024-09-22T10:59:12.058Z] [INFO] [1825229511] game "e79852891683a2f203011c055d9236bb" started (please wait 45 seconds)
[2024-09-22T10:59:57.318Z] [INFO] [1825229511] game "e79852891683a2f203011c055d9236bb" finished +255 pts [4/5]
[2024-09-22T11:00:02.591Z] [INFO] [1825229511] game "d0f6568357650558317dc6563766c77f" started (please wait 45 seconds)
[2024-09-22T11:00:47.805Z] [INFO] [1825229511] game "d0f6568357650558317dc6563766c77f" finished +255 pts [5/5]
[2024-09-22T11:01:01.663Z] [INFO] [1825229511] user authenticated
[2024-09-22T11:01:01.777Z] [WARN] [1825229511] get info (0 tickets left) -> game cancelled
[2024-09-22T11:01:05.221Z] [INFO] [1825229511] user authenticated
[2024-09-22T11:01:05.333Z] [WARN] [1825229511] get info (0 tickets left) -> game cancelled
[2024-09-22T11:01:11.246Z] [INFO] [1825229511] user authenticated
[2024-09-22T11:01:11.350Z] [WARN] [1825229511] get info (0 tickets left) -> game cancelled
[2024-09-22T11:03:29.939Z] [INFO] [1825229511] user authenticated
[2024-09-22T11:03:30.043Z] [WARN] [1825229511] get info (0 tickets left) -> game cancelled
[2024-09-22T11:03:33.300Z] [INFO] [1825229511] user authenticated
[2024-09-22T11:03:33.414Z] [WARN] [1825229511] get info (0 tickets left) -> game cancelled
[2024-09-22T11:03:36.166Z] [INFO] [1825229511] user authenticated
[2024-09-22T11:03:36.277Z] [WARN] [1825229511] get info (0 tickets left) -> game cancelled
[2024-09-22T11:03:38.671Z] [INFO] [1825229511] user authenticated
[2024-09-22T11:03:38.784Z] [WARN] [1825229511] get info (0 tickets left) -> game cancelled
[2024-09-22T11:03:40.118Z] [INFO] [1825229511] user authenticated
[2024-09-22T11:03:40.226Z] [WARN] [1825229511] get info (0 tickets left) -> game cancelled
[2024-09-22T11:03:41.457Z] [INFO] [1825229511] user authenticated
[2024-09-22T11:03:41.567Z] [WARN] [1825229511] get info (0 tickets left) -> game cancelled
[2024-09-22T11:03:43.025Z] [INFO] [1825229511] user authenticated
[2024-09-22T11:03:43.127Z] [WARN] [1825229511] get info (0 tickets left) -> game cancelled
[2024-09-22T11:05:00.363Z] [INFO] [1825229511] user authenticated
[2024-09-22T11:05:00.367Z] [INFO] [1825229511] user authenticated
[2024-09-22T11:05:00.369Z] [INFO] [1825229511] user authenticated
[2024-09-22T11:05:00.393Z] [INFO] [1825229511] user authenticated
[2024-09-22T11:05:00.408Z] [INFO] [1825229511] user authenticated
[2024-09-22T11:05:00.421Z] [INFO] [1825229511] user authenticated
[2024-09-22T11:05:00.430Z] [INFO] [1825229511] user authenticated
[2024-09-22T11:05:00.433Z] [INFO] [1825229511] user authenticated
[2024-09-22T11:05:00.517Z] [INFO] [1825229511] user authenticated
[2024-09-22T11:05:00.536Z] [INFO] [1825229511] user authenticated
[2024-09-22T11:05:00.563Z] [INFO] [1825229511] user authenticated
[2024-09-22T11:05:00.569Z] [INFO] [1825229511] user authenticated
[2024-09-22T11:05:00.674Z] [WARN] [1825229511] get info (0 tickets left) -> game cancelled
[2024-09-22T11:05:03.538Z] [ERROR] [1825229511] Cannot read properties of null (reading 'qualified')
[2024-09-22T11:05:03.548Z] [ERROR] [1825229511] Cannot read properties of null (reading 'qualified')
[2024-09-22T11:05:03.562Z] [ERROR] [1825229511] Cannot read properties of null (reading 'qualified')
[2024-09-22T11:05:03.568Z] [ERROR] [1825229511] Cannot read properties of null (reading 'qualified')
[2024-09-22T11:05:03.568Z] [ERROR] [1825229511] Cannot read properties of null (reading 'qualified')
[2024-09-22T11:05:03.569Z] [ERROR] [1825229511] Cannot read properties of null (reading 'qualified')
[2024-09-22T11:05:03.574Z] [ERROR] [1825229511] Cannot read properties of null (reading 'qualified')
[2024-09-22T11:05:03.583Z] [ERROR] [1825229511] Cannot read properties of null (reading 'qualified')
[2024-09-22T11:05:03.664Z] [ERROR] [1825229511] Cannot read properties of null (reading 'qualified')
[2024-09-22T11:05:03.665Z] [ERROR] [1825229511] Cannot read properties of null (reading 'qualified')
[2024-09-22T11:05:03.670Z] [ERROR] [1825229511] Cannot read properties of null (reading 'qualified')
[2024-09-22T11:05:13.632Z] [INFO] [1825229511] user authenticated
[2024-09-22T11:05:13.729Z] [WARN] [1825229511] get info (0 tickets left) -> game cancelled
[2024-09-22T11:07:22.031Z] [INFO] [1825229511] user authenticated
[2024-09-22T11:07:22.219Z] [INFO] [1825229511] get info (1 tickets left)
[2024-09-22T11:07:24.491Z] [INFO] [1825229511] game "4bf97e5c217c4b0bac4accea8ea4f680" started (please wait 45 seconds)
[2024-09-22T11:08:09.877Z] [INFO] [1825229511] game "4bf97e5c217c4b0bac4accea8ea4f680" finished +325 pts [1/1]
[2024-09-22T11:08:23.635Z] [INFO] [1825229511] user authenticated
[2024-09-22T11:08:23.740Z] [WARN] [1825229511] get info (0 tickets left) -> game cancelled

**Comment by forforgebe** [comment]
Did you perform any action after run auto menu?

**Comment by Araara7** [comment]
> Did you perform any action after run auto menu?

aah im forget sir , let me try again from npm start

**Comment by Araara7** [comment]
sir this bot support for multi account ? 

can i add more query in tg-hash.txt ? example
1. query_id=.....
2. query_id=.....

**Comment by forforgebe** [comment]
> sir this bot support for multi account ?
> 
> can i add more query in tg-hash.txt ? example
> 
> 1. query_id=.....
> 2. query_id=.....

yup it is

**Comment by Araara7** [comment]
sir why bot can't use again ?

**Comment by forforgebe** [comment]
> sir why bot can't use again ?

I disable all user for some reason but if you want to keep use this bot, drop your user here (please make sure your user is censored) I will enable it
